core/model/upgradeseries.go

```diff
@@ -7,103 +7,29 @@ import (
-
-// ToOldStatus is a shim between the new and old status representations.
-// It exists to bootstrap that transition. Unit tests omitted by design.
-func (s UpgradeSeriesStatus) ToOldStatus() (UpgradeSeriesStatusType, MachineSeriesUpgradeStatus) {
-	switch s {
-	case UpgradeSeriesNotStarted:
-		return PrepareStatus, MachineSeriesUpgradeNotStarted
-	case UpgradeSeriesPrepareStarted:
-		return PrepareStatus, MachineSeriesUpgradeStarted
-	case UpgradeSeriesPrepareMachine:
-		return PrepareStatus, MachineSeriesUpgradeAgentsStopped
-	case UpgradeSeriesPrepareComplete:
-		return PrepareStatus, MachineSeriesUpgradeComplete
-	case UpgradeSeriesCompleteStarted:
-		return CompleteStatus, MachineSeriesUpgradeStarted
-	case UpgradeSeriesComplete:
-		return CompleteStatus, MachineSeriesUpgradeComplete
-	default:
-		return "", ""
-	}
-}
-
-func FromOldUpgradeSeriesStatus(prepare, complete MachineSeriesUpgradeStatus) (UpgradeSeriesStatus, error) {
-	switch prepare {
-	case MachineSeriesUpgradeNotStarted:
-		return UpgradeSeriesNotStarted, nil
-	case MachineSeriesUpgradeStarted:
-		return UpgradeSeriesPrepareStarted, nil
-	case MachineSeriesUpgradeAgentsStopped:
-		return UpgradeSeriesPrepareMachine, nil
-	}
-
-	// Arriving here, prepare is complete, so we check the complete status.
-	switch complete {
-	case MachineSeriesUpgradeNotStarted:
-		return UpgradeSeriesPrepareComplete, nil
-	case MachineSeriesUpgradeStarted:
-		return UpgradeSeriesCompleteStarted, nil
-	case MachineSeriesUpgradeComplete:
-		return UpgradeSeriesComplete, nil
-	}
-
-	return "", errors.Errorf("unable to determine status from old combination: %q/%q", prepare, complete)
-}
-
-// The Statuses, at least for units, appy to both the "Prepare" and "Complete"
-// phases of a managed series upgrade. This type can be used to distinguish
-// between those phases when working with the state of an upgraded.
-type UpgradeSeriesStatusType string
-
-const (
-	PrepareStatus  UpgradeSeriesStatusType = "Prepare"
-	CompleteStatus UpgradeSeriesStatusType = "Complete"
-)
-
-//MachineSeriesUpgradeStatus is the current status a machine series upgrade
-type MachineSeriesUpgradeStatus string
-
-const (
-	MachineSeriesUpgradeNotStarted    MachineSeriesUpgradeStatus = "NotStarted"
-	MachineSeriesUpgradeStarted       MachineSeriesUpgradeStatus = "Started"
-	MachineSeriesUpgradeAgentsStopped MachineSeriesUpgradeStatus = "AgentsStopped"
-	MachineSeriesUpgradeComplete      MachineSeriesUpgradeStatus = "Complete"
-)
-
-//UnitSeriesUpgradeStatus is the current status of a series upgrade for units
-type UnitSeriesUpgradeStatus string
+// Machine upgrade series status will be removed
+type MachineSeriesUpgradeStatus = UpgradeSeriesStatus
 
 const (
-	UnitNotStarted UnitSeriesUpgradeStatus = "NotStarted"
-	UnitStarted    UnitSeriesUpgradeStatus = "Started"
-	UnitErrored    UnitSeriesUpgradeStatus = "Errored"
-	UnitCompleted  UnitSeriesUpgradeStatus = "Completed"
+	NotStarted       UpgradeSeriesStatus = "not started"
```

Comment:
I would be inclined to keep these suffixed with "UpgradeSeries" to disambiguate them from other constants that this package may export in the future.